pkg/plugin/plugin.go

```diff
@@ -101,6 +105,14 @@ func EmitEvent(name string, event interface{}) error {
 	return nil
 }
 
+// ValidateConfig validates the plugin configuration data
+func ValidateConfig(name string, rawConfig Config) (bool, error) {
+	if plugin, ok := plugins[name]; ok {
+		return plugin.Validate(rawConfig)
```

Comment:
pls add debug log here, just to know something is happening, and it would be nice to log result and error as well somewhere, e.g. in teh calling code